controllers/suite_test.go

```diff
@@ -118,7 +118,17 @@ func TestAPIs(t *testing.T) {
 
 	suiteCfg, reporterCfg := GinkgoConfiguration()
 	ReconfigureGinkgo(&reporterCfg, &suiteCfg)
-	SetDefaultEventuallyTimeout(time.Second * 5)
+	singleTestTimeout := os.Getenv("SINGLE_TEST_TIMEOUT")
+	if singleTestTimeout != "" {
+		timeout, err := time.ParseDuration(singleTestTimeout)
+		if err != nil {
+			SetDefaultEventuallyTimeout(time.Second * 5)
```

Comment:
nit: might be worth at least printing the error with some info so user knows tests are running with 5s timeout

Reply:
In case there is an error, fail it and print error.